src/components/MenuItem.js

```diff
@@ -46,6 +47,8 @@ const defaultProps = {
     interactive: true,
     fallbackIcon: Expensicons.FallbackAvatar,
     brickRoadIndicator: '',
+    floatRightAvatars: [],
+    shouldStackHorizontally: false,
```

Comment:
`shouldStackHorizontally` is not added in `propTypes`.

Reply:
Updated